### PR TITLE
Document updates

### PR DESCRIPTION
**Pull request by RadhakrishnaJ**
1. Explained what is HPE passport account.
2. Corrected reference link which was pointing to private git. 
3. Corrected folder path in examples.
4. Added License server references in examples. 
5. Corrected typo errors.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Several examples of using Swarm Learning are provided with the package. 
 
 ## Prerequisite for all examples
-Start license server and install valid license before running any of the examples. 
+Start license server and install valid license before running any of the examples. Refer [Installing licenses and starting license server](https://github.com/RadhakrishnaJ/swarm-learning/blob/master/docs/setup.md#installing-licenses-and-starting-license-server).
```
Comment (suresh-ls) [inline examples/README.md:6]: The path has to be to the main/master...  Not to your forked repo...

File: examples/README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ These examples are designed to run in about 8 to 10 machines and demonstrate the
    3. fraud-detection: It runs decentralized training on structured Credit Card Fraud dataset for credit card fraud 
        detection using TensorFlow.
 
-See the README file for each example for brief instructions on running them. For a more detailed reference on the Swarm Learning platform and package, see [Readme](https://github.hpe.com/adc/swarm-learning-docs/blob/extgit/README.md)
+See the README file for each example for brief instructions on running them. For a more detailed reference on the Swarm Learning platform and package, see [Readme](https://github.com/RadhakrishnaJ/swarm-learning#readme)
```
Comment (suresh-ls) [inline examples/README.md:22]: Same issue. The path has to be to the main/master... Not to your forked repo...

File: docs/setup.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 The Swarm Learning package contains docker container images for each Swarm Learning component. The Swarm Learning container images are available in a docker registry as described in section [Pulling docker images](#pull-docker-images).
 
+Create login credentials to access [My HPE Software Center (MSC)](https://myenterpriselicense.hpe.com/cwp-ui/evaluation/HPE-SWARM/0.3.0/null) if you don’t have yet. The mail address used to access MSC is called as HPE passport account. 
+
```
Comment (suresh-ls) [inline docs/setup.md:6]: make it 'email' address

**Review by suresh-ls — APPROVED** [review]
Ok, mostly the cross-ref to the child fork, would be updated to parent repo, post the merge... Go ahead and try :-)

File: examples/README.md (hunk 1 above)
Comment (RadhakrishnaJ) [reply examples/README.md]: Added relative path.

File: examples/README.md (hunk 2 above)
Comment (RadhakrishnaJ) [reply examples/README.md]: Added relative path

File: docs/setup.md (hunk 3 above)
Comment (RadhakrishnaJ) [reply docs/setup.md:6]: updated to use email.

**Review by sharvanideshmukh — APPROVED** [review]

**Pull request merged**